### ISSUE TITLE
Datasets for Sybil/Impact analysis on Arbitrum

### ISSUE THREAD

**Issue by DistributedDoge** [OpenDataforWeb3/DataGrantsforARB#2]
The goal of proposed bounty is to encourage sharing of useful datasets that can aid future Sybil/Grant Analysis. To make end-results easier to compare, I suggest focusing on static datasets that would be assessed according to value they bring to Analyst community.

Datasets produced for this bounty should be well-described so that everyone can understand:

- what data is contained within
- why is it valuable to community
- how and when the dataset was constructed

Example ideas, but the more creative the approach is, the better:

- all smart contracts/EOAs involved in Arbitrum airdrops + event logs or tx info (bonus points if decoded)
- listing of github repositories associated with grants running on Arbitrum (bonus points if has some impact metrics)

Output submitted for judging would be a collection of flat-files (or single-file database like e.g. duckdb) containing information participants managed to gather + description of the dataset.

Bonus points if dataset:

- has value for re-use in future hackathons
- incorporates data coming from multiple sources
- is used by the participant to fulfill another bounty (demonstrating how to use it)

**Comment by Pfed-prog** [comment]
> Output submitted for judging would be a collection of flat-files (or single-file database like e.g. duckdb) containing information participants managed to gather + description of the dataset.

In my humble opinion, ipfs files would also be very great since that would provide a censorship resistant data storage medium

**Comment by ARDev097** [comment]
Hi,
Thanks for creating this issue.
Is it possible to provide some more example ideas?
Also, if we are looking for github repositories associated with grants(means grantee, right?).
And this might be more of a manual task and a static dataset in a db or google sheet, should that be fine?

**Comment by DistributedDoge** [comment]
@ARDev097 If you do manual work collecting interesting data and provide static dataset as google sheet, that would be a valid submission here! 

As `pfed-prog` has suggested, if you also package end result as `.parquet` or `.csv` and then push it to IPFS somewhere (free tier of https://www.pinata.cloud/ could help) that would be even nicer.


**Github data example:**

Check [oss-observer-blog](
https://docs.opensource.observer/blog/arb-ecosystem-analysis) link to see what kind of analysis is possible having Github data. You don't need to do analysis (different bounty linked below) but you can help collect the data.


If you look around this [organization](https://github.com/opensource-observer), you would see they already have some interesting datasets, especially is `oss-directory` and `oss-insights` repository. 

Some of data from `oss-directory` is hard to consume (tiny .yaml files) so if someone turned that into one big table/spreadsheet with information about all projects in Arbitrum Ecosystem I would find such submission valuable. Just remember our interest is only projects in Arbitrum ecosystem, not broader open-source.

**Manual collection example:**

Another example well-suited for manual data-gathering is to extend work done by those folks to include all kinds of Arbitrum grants, not just STIP grant they are already tracking:

https://github.com/andrewhong5297/Crypto-Grants-Analysis/blob/main/uploads/evm_grants.csv

**Similar bounties:**

- If you feel like tracking arbitrum grantees across multiple ecosystems, your work can also qualify to [cross-ecosystem](https://github.com/OpenDataforWeb3/DataGrantsforARB/issues/8) bounty.
- If parts of your process can be automated (which you don't need for **this** bounty) you can submit code to [pipeline](https://github.com/OpenDataforWeb3/DataGrantsforARB/issues/11) bounty. 
- If you gather Github data on projects, you can proceed with [analysis](https://github.com/OpenDataforWeb3/DataGrantsforARB/issues/13) here. Or someone else can do analysis using your data!

**Tips:**
- If you feel like what you are doing is on edge of two bounties, feel free to submit your work to both and we will sort it out. 
- If I may suggest something - look at bounties that we have tagged `Analysis - for bounty vote` and ask yourself **what kind of data can I collect to make it easy for me (or sb else) to do that bounty?**

**Comment by ARDev097** [comment]
Thanks for the reply above.
I started understanding more about oss-directory and oss-insights repository and have checked the structure in .yaml files and currenty I am in the final stage on getting the spreadsheet with information about all projects in Arbitrum Ecosystem.

**Comment by ARDev097** [comment]
"Hi, I have worked on this part
'If you look around this organization, you would see they already have some interesting datasets, especially is oss-directory and oss-insights repository.
Some of data from oss-directory is hard to consume (tiny .yaml files) so if someone turned that into one big table/spreadsheet with information about all projects in Arbitrum Ecosystem I would find such submission valuable. Just remember our interest is only projects in Arbitrum ecosystem, not broader open-source.'
Merged all the .yaml files and have created three different files,
All protocols across different ecosystems. - [all_ecosystem_data.csv](https://coffee-cheap-marten-178.mypinata.cloud/ipfs/QmUxeaREVVi3rM8DKnyg8kNdAuHQkwTBSYUfYt4mVQG7H3)
All protocols on Arbitrum ecosystem. - [arbitrum_all_protocols_data (Distributed).csv](https://coffee-cheap-marten-178.mypinata.cloud/ipfs/QmRebupVUL3tssT3BdX4jVnY5ZywA9eHaE1dt2QSPUX7jL)
All protocols in Arbitrum with unique row for each protocol. - [arbitrum_unique_protocols_data (Unique row for each protocol).csv](https://coffee-cheap-marten-178.mypinata.cloud/ipfs/QmPLRJcZsAhPQwwmiXqrRaC4wo1EEpRAf4H6uJAeGfF28h)
All three csv file explanation - [yaml_task.md](https://coffee-cheap-marten-178.mypinata.cloud/ipfs/QmavHhnWg5NrgpjsJAnVwxjMa7JuEdspwYLwMmaw72W1B3)

Thank you for the help.
Should I submit it on DeWork or over here is fine?"
